intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/MinorWarnings.kt

```diff
@@ -169,3 +170,16 @@ class IllegalPluginIdPrefix(private val illegalPluginId: String, private val ill
     get() = "Plugin ID '$illegalPluginId' has an illegal prefix '$illegalPrefix'. See https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html#idea-plugin__id"
 }
 
+class ServiceExtensionPointPreloadNotSupported(private val serviceType: IdePluginContentDescriptor.ServiceType) : PluginProblem() {
+  override val level
+    get() = Level.WARNING
+
+  override val message
+    get() = "Service preloading in the 'preload' attribute is deprecated in the <${serviceType.toXmlElement()}> element. Consider migrating to the listeners. See https://plugins.jetbrains.com/docs/intellij/plugin-listeners.html"
```

Comment:
Service preloading is deprecated in the ...

Consider migrating to listeners, see ...

Reply:
I'll leave the 'preload' attribute in the message. It might be not obvious what to change just from the message.

Reply:
ok